docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerCloud.java

```diff
@@ -269,10 +265,11 @@ public Node call() throws Exception {
                                     //
                                     // deferring the completion of provisioning until the launch
```

Comment:
This is the wrong way to wait. What you want to do is check the docker container until it is **ready** to connect and then **and only then** return. This way sadly needs to do the evil adding to Jenkins so that `toComputer()` does not return `null`...

wrong wrong and more wrong!